common/web/lm-worker/build-wrap-and-minify.js

```diff
@@ -34,7 +34,7 @@ if(MINIFY) {
     sourcemap: 'external',
     sourcesContent: DEBUG,
     minify: true,
-    keepNames: true,
+    keepNames: false, // Do NOT enable - will break under Android 5.0 / Chrome 35 environments!
```

Comment:
An update thanks to https://github.com/evanw/esbuild/issues/3477#issuecomment-1795519157: this is likely a non-issue starting in Chrome 42.

https://caniuse.com/mdn-javascript_builtins_function_name_configurable_true